packages/mobile/e2e/src/usecases/Support.js

```diff
@@ -0,0 +1,50 @@
+import { dismissBanners } from '../utils/banners'
+import { pixelDiff, scrollIntoView } from '../utils/utils'
+
+export default Support = () => {
+  beforeEach(async () => {
+    await device.reloadReactNative()
+    await dismissBanners()
+  })
+
+  if (device.getPlatform() === 'ios') {
+    it("Display 'Contact' on Shake", async () => {
```

Comment:
mixed used of `"` and `'` -- i'm surprised a linter / styler doesn't catch this.  Should it?

Reply:
Ideally it would be `'Display \'Contact\' on Shake'`, but VScode reformats to above regardless of my VScode format on save options.